packages/yoga/src/Stepper/web/Stepper.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { limitChildren, typeOf } from '../../shared';
+import Dots from './Dots';
+import Line from './Line';
+
+const Root = styled.div`
+  width: 100%;
+`;
+
+const LineWrapper = styled.div(
+  ({ theme: { spacing } }) => `
+  width: 100%;
+  height: 46px;
+
+  padding: 0 ${spacing.xxlarge}px;
```

Comment:
this padding should use the component token, not the theme token right?

from:
```
 ${spacing.xxlarge}
```

to: 
```
${stepper.padding.left}
and
${stepper.padding.right}
```